packages/shared/components/molecules/activity-info/NftMetadataInformation.svelte

```diff
@@ -0,0 +1,80 @@
+    import { Activity, NftActivity } from '@core/wallet'
+    import { selectedAccountIndex } from '@core/account'
+
+    export let activity: Partial<Activity> = {}
+    export let nftMetadata: Partial<IIrc27Metadata> = {}
+
+    type NftMetadataDetailsList = {
+        [key in keyof IIrc27Metadata]: {
+            data: unknown
+            isTooltipVisible?: boolean
+        }
+    }
+
+    $: storedNft = getNftByIdFromAllAccountNfts($selectedAccountIndex, (activity as NftActivity)?.nftId)
+    $: nftMetadataDetailsList = createNftMetadataDetailsList(
+        storedNft?.parsedMetadata ?? (nftMetadata as IIrc27Metadata)
+    )
+
+    function createNftMetadataDetailsList(
+        metadata: IIrc27Metadata | string
+    ): NftMetadataDetailsList | { metadata: { data: string } } {
+        if (typeof metadata === 'string') {
+            return { metadata: { data: metadata } }
+        }
+        return createIrc27NftMetadataDetailsList(metadata)
+    }
+
+    function createIrc27NftMetadataDetailsList(metadata: IIrc27Metadata): NftMetadataDetailsList {
+        return {
+            ...(metadata?.standard && {
+                standard: { data: metadata.standard, isTooltipVisible: true },
+            }),
+            ...(metadata?.version && {
+                version: { data: metadata.version },
+            }),
+            ...(metadata?.name && {
+                name: { data: metadata.name },
+            }),
+            ...(metadata?.type && {
+                type: { data: metadata.type as string, isTooltipVisible: true },
+            }),
+            ...(metadata?.uri && {
+                uri: { data: metadata.uri },
+            }),
+            ...(metadata?.collectionId && {
+                collectionId: { data: metadata.collectionId, isTooltipVisible: true },
+            }),
+            ...(metadata?.collectionName && {
+                collectionName: { data: metadata.collectionName },
+            }),
+            ...(metadata?.royalties && {
+                royalties: { data: metadata.royalties, isTooltipVisible: true },
+            }),
+            ...(metadata?.issuerName && {
+                issuerName: { data: metadata.issuerName, isTooltipVisible: true },
+            }),
+            ...(metadata?.description && {
+                description: { data: metadata.description },
+            }),
+            ...(metadata?.attributes && {
+                attributes: { data: metadata.attributes, isTooltipVisible: true },
+            }),
+        }
+    }
+</script>
+
+{#each Object.entries(nftMetadataDetailsList) as [key, value]}
+    <KeyValueBox
+        keyText={localize(`views.collectibles.metadata.${key}`)}
+        valueText={value.data}
+        tooltipText={value.isTooltipVisible ? localize(`tooltips.transactionDetails.nftMetadata.${key}`) : undefined}
+        classes={key === 'metadata' ? 'whitespace-pre-wrap' : ''}
+        isCopyable
```

Comment:
not sure if we want every field to be copyable. Clarify that with nicole/UX

Reply:
Clarified with Nicole, and we both agree that every field should be copyable